UserAPI/Login.md

```diff
@@ -1,3 +1,46 @@
 # Login
 
-To login all you need to do is call any API with the Authorization header and save the `auth` cookie returned by the server.
\ No newline at end of file
+This API allows you to get user data from a VRChat account
+
+## Request Method
+GET
+
+## Endpoint
+    https://api.vrchat.cloud/api/1/auth/user
+
+## Requires Authentication
+Yes
+
+## Returns
+Field | Type | Description
+------|------|------------
+id|string|The user's unique identifier
+username|string|The user's username
+displayName|string|The user's display name
+pastDisplayName|string[]|The user's past display names
+hasEmail|bool|True if the user have a verified email address
+hasPendingEmail|bool|True if the user have an unverified email address
+obfuscatedEmail|string|The obfuscated email of the user
+obfuscatedPendingEmail|string|The obfuscated pending email of the user
+emailVerified|bool|True if the user have verified his email address
+hasBirthday|bool|True if the user have his birthday set
+unsubscribe|bool|True if the user have unsubscribed to the VRChat newsletter
+friends|string[]|An array of the user's friends userId
+friendGroupNames|string[]|
+currentAvatarImageUrl|string|The url to the weared avatar image
+currentAvatarThumbnailImageUrl|string|The url to the weared avatar thumbnail
+currentAvatar|string|The weared avatar id
+currentAvatarAssetUrl|string|The url to the weared avatar asset
+status|string|The current user's status
+statusDescription|string|The current user's status description
+acceptedTOSVersion|string|The last accepted TOS version
+steamDetails|Steam Details|The steam details. Empty when using a VRChat account
+hasLoggedInFromClient|bool|Return true if the client use a steam account
```

Comment:
This is set to false when logging in on the website, and true when logging in from the client. A more suiting description would be "Whether the user is logging in from the client", or similar